### DISCUSSION TITLE
stable version is not avaliable for linux via package managers, link is invalid

### DISCUSSION THREAD

**Discussion by Ashish-CodeJourney** [zed-industries/zed#14043] [Feature Requests]
![Screenshot from 2024-07-10 11-30-09](https://github.com/zed-industries/zed/assets/102389071/16cd3817-db78-41bc-b6dd-dfffab026be7)

links of stable versions should be removed

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
